[2024-11-05T07:29:11.743+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-05T07:29:11.791+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task scheduled__2024-11-04T00:00:00+00:00 [queued]>
[2024-11-05T07:29:11.826+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task scheduled__2024-11-04T00:00:00+00:00 [queued]>
[2024-11-05T07:29:11.827+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-11-05T07:29:11.906+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): third_task> on 2024-11-04 00:00:00+00:00
[2024-11-05T07:29:11.915+0000] {standard_task_runner.py:72} INFO - Started process 344 to run task
[2024-11-05T07:29:11.922+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v3', 'third_task', 'scheduled__2024-11-04T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/our-first-dag.py', '--cfg-path', '/tmp/tmpxtuz4lie']
[2024-11-05T07:29:11.926+0000] {standard_task_runner.py:105} INFO - Job 27: Subtask third_task
[2024-11-05T07:29:12.092+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v3.third_task scheduled__2024-11-04T00:00:00+00:00 [running]> on host f4d9f79457ef
[2024-11-05T07:29:12.326+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Adithya' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-04T00:00:00+00:00'
[2024-11-05T07:29:12.327+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-05T07:29:12.364+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-05T07:29:12.368+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello I am after task2']
[2024-11-05T07:29:12.388+0000] {subprocess.py:86} INFO - Output:
[2024-11-05T07:29:12.391+0000] {subprocess.py:93} INFO - hello I am after task2
[2024-11-05T07:29:12.393+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-05T07:29:12.488+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-05T07:29:12.490+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=third_task, run_id=scheduled__2024-11-04T00:00:00+00:00, execution_date=20241104T000000, start_date=20241105T072911, end_date=20241105T072912
[2024-11-05T07:29:12.584+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-05T07:29:12.627+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-05T07:29:12.631+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
